src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -74,8 +74,6 @@ internal sealed class ActiveSolutionBoundTracker : IActiveSolutionBoundTracker,
             this.errorListInfoBarController = this.extensionHost.GetService<IErrorListInfoBarController>();
             this.errorListInfoBarController.AssertLocalServiceIsNotNull();
 
-            //TODO: check whether the errorListInfobarController needs to be refreshed
```

Comment:
I've debugged through when opening/closing solutions and opening/closing VS with/without an initial solution and the error list controller is correctly refreshed in each case.